README.md

```diff
@@ -1,104 +1,153 @@
 
-```
+  EX)
 
-### Question #2
+  function Bike(model, color) {
+    this.model = model;
+    this.color = color;
+  }
 
-Define a Javascript constructor called 'Instructor'. Every instance of Instructor should have a `name` property, and a method called `givesHomework`. This method takes one argument called `assignment` and, when executed, console-logs "[name] gives the students [assignment] for Friday's homework."
+  Bike.prototype.ride = function() {
+    console.log("Yeha!");
+  }
 
-Instantiate an instructor named 'Robin' and call its `givesHomework` method with "Intro to Ruby" as the argument.
+  var Max = new Bike("MTX", silver);
 
-Your Answer:
+  ```
 
-```js
-// your code here
+  ### Question #2
 
-```
-### Question #3
+  Define a Javascript constructor called 'Instructor'. Every instance of Instructor should have a `name` property, and a method called `givesHomework`. This method takes one argument called `assignment` and, when executed, console-logs "[name] gives the students [assignment] for Friday's homework."
 
-Given the following front-end JS model, add an instance method for all pandas called `eat_bamboo`. Calling this method should increase that panda's value for `num_bamboo_eaten` by 1.
+  Instantiate an instructor named 'Robin' and call its `givesHomework` method with "Intro to Ruby" as the argument.
 
-```js
+  Your Answer:
 
-var Panda = function(name, age) {
-  this.name = name;
-  this.age = age;
-  this.num_bamboo_eaten = 0;
-}
-```
-Your Answer:
-```js
-// your code here
-```
+  ```js
+  function Instructor(name) {
+    this.name = name;
+    this.givesHomework = function(assignment){
+      console.log(name + "gives the students" + assignment + "for Friday's homework.")
+    }
+  }
 
-### Question #4
+  var robin = new Instructor("Robin");
+  robin.givesHomework("Intro to Ruby");
 
-Describe the importance of using object-oriented programming.
 
-Your Answer:
-```js
-// your answer here
-```
 
-## jQuery
+  ```
+  ### Question #3
 
-### Question #5
+  Given the following front-end JS model, add an instance method for all pandas called `eat_bamboo`. Calling this method should increase that panda's value for `num_bamboo_eaten` by 1.
 
-Which of the following statements will work, assuming jQuery is loaded?
+  ```js
 
-Select all that apply:
-```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.querySelectorAll(".post")[0].innerHTML`
-[] `document.querySelectorAll(".post").innerHTML`
-```
+  var Panda = function(name, age) {
+    this.name = name;
+    this.age = age;
+    this.num_bamboo_eaten = 0;
+  }
+  ```
+  Your Answer:
+  ```js
 
-### Question #6
+  Panda.prototype.eat_bamboo = function() {
+    this.num_bamboo_eaten = num_bamboo_eaten + 1;
```

Comment:
You'd need another `this.` :  `this.num_bamboo_eaten = this.num_bamboo_eaten + 1`.